lib/client/api.go

```diff
@@ -94,9 +94,6 @@ type Config struct {
 	// or use any local certs, and not use interactive logins
 	SkipLocalAuth bool
 
-	// AuthMethods if passed will be used to login into cluster
-	AuthMethods []ssh.AuthMethod
-
```

Comment:
Can you bring it back? I use it in the library mode, when passing external authenticators (not local auth)